File: src/php/StringManipulation.php
<?php

namespace EntityFX\NetBenchmark\Core\Generic {

    use EntityFX\NetBenchmark\Core\Writer;
    
    class StringManipulation  extends StringManipulationBase {

        protected function BenchImplementation()
        {
            $str = "the quick brown fox jumps over the lazy dog";
            $str1 = "";
            for ($i = 0; $i < $this->Iterrations; $i++)
            {
                $str1 = self::DoStringManipilation($str);
            }
            return $str1;
        }
    }
}

File: src/dotnet/EntityFX.NetBenchamarks.Core/Generic/StringManipulation.cs
using System.Diagnostics;

namespace EntityFX.NetBenchmark.Core.Generic
{
    public class StringManipulation : StringManipulationBase<string>, IBenchamrk
    {
        public StringManipulation(IWriter writer) : base(writer)
        {
        }

        public override string BenchImplementation()
        {
            var str = "the quick brown fox jumps over the lazy dog";
            string str1 = string.Empty;
            for (int i = 0; i < Iterrations; i++)
            {
                str1 = DoStringManipilation(str);
            }
            return str1;
        }
    }
}
